components/Button/Button.jsx

```diff
@@ -59,10 +59,10 @@ const StyledButton = styled.button`
     },
   }) => {
     const paddings = {
```

Comment:
The papding values should not be multiples of two?

Reply:
And it should be getting from the theme

Reply:
A solution has been proposed in the last commit.